txpool/tx_pool.go

```diff
@@ -307,6 +309,7 @@ func (p *TxPool) StrictlyAdd(newTx *tx.Transaction) error {
 // Remove removes tx from pool by its Hash.
 func (p *TxPool) Remove(txHash thor.Bytes32, txID thor.Bytes32) bool {
 	if p.all.RemoveByHash(txHash) {
+		metricTxPoolGauge().GaugeWithLabel(-1, map[string]string{"source": "n/a", "total": "true"})
```

Comment:
What does total mean ?

Reply:
Good q ! The total tag allows to cumulative add the metric. Let's say you want to plot how many tx's are there in the mempool, it's possible to use total. And it's possible to breakdown the mempool txs by if they arrived remotely or via local api.

Added a comment in the code 👍